.gitlab-ci.yml

```diff
@@ -303,6 +305,10 @@ integration: # merging all artifacts to produce single R repository and summarie
     - Rscript -e 'pdf.copy("data.table", "test-rel-lin")'
     # web/checks/check_results_$pkg.html
     - Rscript -e 'check.index("data.table", names(test.jobs))'
+    # pkgdown
+    - apt-get update -qq && apt-get install -y libxml2-dev
+    - Rscript -e 'install.packages("remotes", repos=Sys.getenv("CRAN_MIRROR"), quiet=TRUE); remotes::install_github("r-lib/pkgdown", repos=Sys.getenv("CRAN_MIRROR"), quiet=TRUE); install.packages("data.table", repos=file.path("file:",normalizePath("bus/integration/cran")), quiet=TRUE); pkgdown::build_site(devel=FALSE, new_process=TRUE, override=list(destination="./pkgdown"))'
```

Comment:
We need to speed it up, currently it installs `remotes`, `pkgdown` and 59! other packages. Plus it requires OS dependency libxml2-dev. As a result this job takes 16 minutes while previously took 5 minutes.

Reply:
GitLab caching is currently in use when mirror suggested dependencies. We do publish all of them together with devel data.table in our R repository, to achieve ultimate reproducibility.
As of now we do not need to use any other package in the pipeline, all CI is in base R. Thus adding 60 packages related to CI only is pretty big thing. We need to make a secondary CI-related cache/repo.
Just for reference, `mirror-packages` job:
https://github.com/Rdatatable/data.table/blob/81af9b6dd7522d91ea86d9e944f6b43a7d21c022/.gitlab-ci.yml#L20-L37

Reply:
after recent commits to pkgdown we went down to 13 minutes for this job
so 3/11 of the overhead is fixed